lib/cartopy/mpl/geoaxes.py

```diff
@@ -1783,9 +1783,7 @@ def _wrap_args(self, *args, **kwargs):
         Handle the interpolation when a wrap could be involved with
         the data coordinates before passing on to Matplotlib.
         """
-        # The shading keyword argument was added in MPL 3.3, so keep
-        # this default updating until we only support MPL>=3.3
-        default_shading = mpl.rcParams.get('pcolor.shading', 'auto')
+        default_shading = mpl.rcParams.get('pcolor.shading')
```

Comment:
I'm not sure how to read the above comment... Can we get rid of the default altogether now and assume kwargs contains "shading" now?

```suggestion
```

Reply:
I think the user can just not pass _shading_, or am I misunderstanding the question?